edb/server/ha/adaptive.py

```diff
@@ -21,11 +21,14 @@
 import asyncio
 import enum
 import logging
+import os
 
 from . import base
 
 
-UNHEALTHY_MIN_TIME = 30
+UNHEALTHY_MIN_TIME = int(os.getenv(
+    'EDGEDB_BACKEND_ADAPTIVE_HA_UNHEALTHY_MIN_TIME', 30
```

Comment:
Is this a public API?

Reply:
I see - the value itself is not test-only, so let me create a new PR to document about the HA support. What about `docs/internals/backend_ha.rst`?

Reply:
> I see - the value itself is not test-only

Then use the `EDGEDB_SERVER` prefix? I'm confused... How is that the variable is for the tests but its value isn't?

>  What about docs/internals/backend_ha.rst?

Sure